synapse/tests/test_axon.py

```diff
@@ -507,15 +507,17 @@ def test_axon_host_spinbackup(self):
                 busurl = 'tcp://127.0.0.1:{}/axonbus'.format(port)
                 hstcfg['vars']['hcfg0']['axon:axonbus'] = busurl
                 hstcfg['vars']['hcfg1']['axon:axonbus'] = busurl
+                svcbus = s_service.openurl('tcp://127.0.0.1:0/axonbus', port=port)  # type: s_service.SvcProxy
 
                 with s_daemon.Daemon() as axondmon:
+                    w = svcbus.waiter(6, 'syn:svc:init')
                     axondmon.loadDmonConf(hstcfg)
-                    svcbus = s_service.openurl('tcp://127.0.0.1:0/axonbus', port=port)  # type: s_service.SvcProxy
-                    time.sleep(2)
+                    w.wait(60)
```

Comment:
To avoid this guess and check cycle, can we just time how long this is actually going to take on the build servers?

Reply:
Alternatively, we could just set these very high (something like 10 minutes) and add a task to investigate and improve axon spinup performance

Reply:
The extremely slow performance was caused by another issue, disregard.